pkg/inject/inject.go

```diff
@@ -1116,7 +1116,7 @@ func (conf *ResourceConfig) InjectNamespace(annotations map[string]string) ([]by
 	return yaml.JSONToYAML(j)
 }
 
-//getFilteredJSON method performs JSON marshaling such that zero values of
+//getFilteredJSON method performs JSON marshalling such that zero values of
```

Comment:
There are various opinions about `l`s in marshal*, the set I've picked seem to be somewhat dominant in general and in this repository.